backend/models/inventoryModel.js

```diff
@@ -18,6 +18,16 @@ class Inventory {
         return err.message;
       }
     }
+    static async addEntry(item, cost, photo, stock) {
+      const query = `INSERT INTO inventory ( item, cost, photo, stock) VALUES ('${item}', ${cost}, '${photo}', ${stock})`;
+  

```

Comment:
Instead of using the cash curly identifier, use $1, $2, $3 in the VALUES for the SQL statement.